ttyplot.c

```diff
@@ -289,9 +290,108 @@ void redraw_screen(const char * errstr) {
     refresh();
 }
 
+// Return a pointer to the last occurrence within [s, s+n) of one of the bytes in the string accept.
+char *find_last(const char *s, size_t n, const char *accept)
+{
+    int pos;
+    for (pos = n - 1; pos >= 0; pos--) {
```

Comment:
Could you merge declaration and definition for `pos` into one?

Reply:
Oh, yes, I missed that! This is a modified version of the previous implementation, where the index was needed after the loop. I did not notice that now `pos` is only needed within the loop. Fixed.